README.md

```diff
@@ -1,20 +1,27 @@
-# Project Name
+# 01-FCC-CAT-PHOTO-APP
 
-> One paragraph statement about the project.
+> A CAT PHOTO APP BUILT USING HTML.
```

Comment:
since it is not an heading, it would look better if you  change the sentence  to a lower case